openlibrary/templates/lib/nav_head.html

```diff
@@ -6,16 +6,16 @@
     <div class="hamburger-dropdown-component">
       <ul class="dropdown-menu hamburger-dropdown-menu">
         $if not ctx.user:
-          <li><a href="/account/login">Log in</a></li>
-          <li><a href="/account/create">Sign up</a></li>
+          <li><a href="/account/login">$_("Log in")</a></li>
+          <li><a href="/account/create">$_("Sign up")</a></li>
 
-        <li><a href="/books/add">Add a Book</a></li>
-        <li><a href="https://archive.org/search.php?sin=TXT">Full-Text Search on Archive.org</a></li>
-        <li><a href="/random">Random Book</a></li>
-        <li><a href="/recentchanges">Recent Community Edits</a></li>
-        <li><a href="/advancedsearch">Advanced Search</a></li>
-        <li><a href="/developers">Developer Center</a></li>
-        <li><a href="/help">Help &amp; Support</a></li>
+        <li><a href="/books/add">$_("Add a Book")</a></li>
+        <li><a href="https://archive.org/search.php?sin=TXT">$_("Full-Text Search on Archive.org")</a></li>
+        <li><a href="/random">$_("Random Book")</a></li>
+        <li><a href="/recentchanges">$_("Recent Community Edits")</a></li>
+        <li><a href="/advancedsearch">$_("Advanced Search")</a></li>
+        <li><a href="/developers">$_("Developer Center")</a></li>
+        <li><a href="/help">$_("Help & Support")</a></li>
```

Comment:
Are we sure we want to go from &amp; -> &?

Reply:
I think it is fine (and preferable here) since this is a python string being passed to `gettext()` and the encoding is done there. We want to match strings on '&' 

Now I have checked, I am confused as to exactly where the encoding takes place, but in the page source I see `<li><a href="/help">Help &amp; Support</a></li>`